app/shopify.server.ts

```diff
@@ -19,17 +18,6 @@ const shopify = shopifyApp({
   sessionStorage: new PrismaSessionStorage(prisma),
   distribution: AppDistribution.AppStore,
   restResources,
-  webhooks: {
-    APP_UNINSTALLED: {
-      deliveryMethod: DeliveryMethod.Http,
-      callbackUrl: "/webhooks",
-    },
-  },
-  hooks: {
-    afterAuth: async ({ session }) => {
-      shopify.registerWebhooks({ session });
-    },
-  },
```

Comment:
I think we can just remove it. It simplifies the config and I assume apps using declarative hooks will be unlikely to use imperative as well - especially since currently the interaction between the two isn't going to work perfectly because we'll delete the declarative events when registering the imperative ones.

Reply:
We can always come back to this and add some documentation if we find folks are running into problems!